dts/bindings/mtd/jedec,spi-nor-common.yaml

```diff
@@ -95,3 +95,15 @@ properties:
 
       This value, when present, identifies bits in the status register
       that should be cleared when the device is initialized.
+
+  mxicy,lh-switch:
```

Comment:
This name is a bit cryptic. I know it follows the name of the corresponding bit ("L/H Switch") from the chip documentation but I guess it can be unintuitive to users. And since this option is MX25R specific, it would be good to reflect this in its name. How about something like "mxicy,mx24r-mode"?

Reply:
I can add `mx25r-`, but I think `mode` is too generic. Is `mx25r-lh-switch` ok? Then it's clear it's only supported on mx25r, so it should be easy to find more info in the datasheet. Other ideas: `mxicy,mx25r-perf-mode`, `mxicy,mx25r-power-mode`. I'm afraid it can be more difficult for the user to understand what this maps to in the datasheet though.

Reply:
Agreed, `mode` alone is too generic. None of the above alternatives is perfect, but to me, `mxicy,mx25r-power-mode` seems to be the least unclear.